here-naksha-lib-core/src/main/java/com/here/naksha/lib/core/models/features/Extension.java

```diff
@@ -65,16 +62,12 @@ public Extension(
       @JsonProperty(URL) @NotNull String url,
       @JsonProperty(VERSION) @NotNull String version,
       @JsonProperty(INIT_CLASS_NAME) @Nullable String initClassName) {
-    this.extensionId = extensionId;
+    super(extensionId);
```

Comment:
Java variable can be `extensionId`, not a problem. But JSON property name will be "id" and not "extensionId". Pls use `XyzFeature.ID` instead of `EXTENSION_ID` (we don't need EXTENSION_ID anymore, pls remove).

Reply:
fixed